6-pubsub/worker.yaml

```diff
@@ -0,0 +1,8 @@
+#runtime: php
+vm: true
+service: worker
+entrypoint: php src/PubSub/entrypoint.php
```

Comment:
Nit: This means no cron daemon will run. The log file may become big. or is it now handled by the host machine now?